client/src/components/Widget/TrackWidget.tsx

```diff
@@ -102,76 +80,83 @@ const TrackWidget = () => {
         <div
           className={css`
             display: flex;
+            border: var(--mi-border);
+            flex-direction: column;
+            width: 100%;
             padding: 1rem;
-            background: var(--mi-normal-background-color);
-            border-radius: 1rem;
-            align-items: center;
-            height: 100%;
+            border-radius: 0.3rem;
             box-sizing: border-box;
+            height: 100%;
+            background: var(--mi-normal-background-color);
           `}
         >
-          <MetaCard
-            title={`${track.title} by ${
-              track.trackGroup.artist?.name ?? "Unknown"
-            }`}
-            description={track.trackGroup.title}
-            image={track.trackGroup.cover?.sizes?.[120]}
-            player={widgetUrl(track.id)}
-          />
-          <ImageWithPlaceholder
-            src={track.trackGroup.cover?.sizes?.[120] ?? ""}
-            alt={track.title}
-            size={120}
-          />
+          <div
+            className={css`
+              display: flex;
+              align-items: center;
+            `}
+          >
+            <MetaCard
```

Comment:
I feel like MetaCard should be pulled to the top of the component, so that we know it's there